packages/react-composites/tests/browser/chat/ChatComposite.test.ts

```diff
@@ -94,35 +94,30 @@ test.describe('Chat Composite E2E Tests', () => {
   });
 
   test('page[1] can rejoin the chat', async ({ pages }) => {
-    const page = pages[1];
-    await page.bringToFront();
-    await page.type(dataUiId(IDS.sendboxTextfield), 'How the turn tables');
-    await page.keyboard.press('Enter');
+    const page1 = pages[1];
+    await page1.bringToFront();
+    await page1.type(dataUiId(IDS.sendboxTextfield), 'How the turn tables');
+    await page1.keyboard.press('Enter');
     // Read the message to generate stable result
     await pages[0].bringToFront();
     await pages[0].waitForSelector(`[data-ui-status="delivered"]`);
 
-    await page.bringToFront();
-    await page.waitForSelector(`[data-ui-status="seen"]`);
-    page.reload({ waitUntil: 'networkidle' });
-    await waitForChatCompositeToLoad(page);
+    await page1.bringToFront();
+    await page1.waitForSelector(`[data-ui-status="seen"]`);
+    await page1.reload({ waitUntil: 'networkidle' });
+    await waitForChatCompositeToLoad(page1);
     // Fixme: We don't pull readReceipt when initial the chat again, this should be fixed in composite
-    await page.waitForSelector(`[data-ui-status="delivered"]`);
-    stubMessageTimestamps(page);
-    expect(await page.screenshot()).toMatchSnapshot('rejoin-thread.png');
-  });
-
-  test.afterAll(async ({ pages }) => {
```

Comment:
We might still need to keep this afterAll - sometimes it just doesn't clean up well, and the process will stay in background without notice, that could cause some memory issues if we don't close all the pages

Reply:
Ohhh I got why you remove this, then we should find a way to close all pages after all the test files running

Reply:
My assumption is once the process ends the browser contexts will be disposed - but can't find documentation on this either way. Did you hit an error before where the browser/pages weren't being killed?